pkg/network/service.go

```diff
@@ -39,6 +40,7 @@ type dbAdapter interface {
 	First(interface{}, ...interface{}) error
 	Create(interface{}) error
 	Delete(interface{}, ...interface{}) error
+	PrintTables()
```

Comment:
Since the 'real' DB driver and its abstraction are not capable of this function, its not a good idea to require it here. To debug the MockDB you may use PrintTables but only in your test package directly with the instance of MockDB you are using.